pkg/credentialstores/common.go

```diff
@@ -0,0 +1,129 @@
+package credentialstores
+
+import (
+	"net/url"
+	"strings"
+
+	"github.com/openshift/image-registry/pkg/kubernetes-common/credentialprovider"
+
+	"github.com/golang/glog"
+)
+
+// MultiError is a wrap for multiple errors.
+type MultiError struct {
```

Comment:
We have used https://github.com/kubernetes/apimachinery/blob/master/pkg/util/errors/errors.go other places

Reply:
nice catch. addressed.